woocommerce/woocommerce-admin
# #1981 Fix/1981 welcome note added

Fixes #1981 

As described I have added a welcome not on plugin installed hook. Let me know if you have any suggestion to improve my PR.

### Screenshots
![new feature](https://user-images.githubusercontent.com/24435113/55630006-40989180-57d2-11e9-8fa9-45fa5f851c2d.png)

### Detailed test instructions:

- Install WooCommerce Admin Plugin
- Check Inbox.
- You will see the new feature's note
